app/src/main/java/org/simple/clinic/storage/migrations/Migration_78.kt

```diff
@@ -0,0 +1,18 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("Classname")
+class Migration_78 @Inject constructor() : Migration(77, 78) {
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.inTransaction {
+      execSQL("""
+        ALTER TABLE "TeleconsultRecord" ADD COLUMN "syncStatus" TEXT
```

Comment:
If you try to run the migration test locally, it will fail since the database schema is now different. The version in the `78.json` schema file has the column as `NOT NULL` but the migration does not add that qualifier.

We should be setting the default value as `PENDING` to existing records as part of the migration.

Reply:
Updated to include `PENDING` as a default not null value in the migration. PTAL